### DISCUSSION TITLE
Хозяин стаи

### DISCUSSION THREAD

**Discussion by Kor-Angar** [dosinabox/g2nr_unofficial_update#426] [General]
> В оригинале Г2НВ тоже так можно, вроде.

ни разу не слышал ничего такого. ну и сам не встречал. это же целая механика игровая: можно стать "лидером" стаи и повести её в атаку куда нибудь. при этом опыт начисляют если даже убивает не герой.

**Comment by Kor-Angar** [comment]
очень крутая штука конечно. видимо всё это тянется ещё с альфы первой готики. есть ещё фича что после удара мракориса орки убегают :-)

**Comment by Kor-Angar** [comment]
ветка с видосиками

**Reply by Kor-Angar** [reply]
Готика 2 классик, тёмный лес в хориносе https://www.youtube.com/watch?v=V3NKzrXfGuM

**Reply by Kor-Angar** [reply]
Архалос https://www.youtube.com/watch?v=IEjoRYPR6Uo

**Comment by Kor-Angar** [comment]
Потестил в классике чуть чуть. в целом работает. 

если(или когда) будет время подскажи где это в коде регулируется, хочется посмотреть в скриптах MDK https://github.com/Kor-Angar/G2_NotR_MDK/tree/master/PrjGOTHIC  что бы из примечаний пираний понять что они хотели сделать :-)

**Reply by dosinabox** [reply]
Возможно, тут: https://github.com/Kor-Angar/G2_NotR_MDK/blob/master/PrjGOTHIC/AI/Monster/B_Monster/B_MM_AssessOthersDamage.d#L50
Но нужно проверять. Может и в обновлении работает, а отключали только для кабанов.

**Reply by dosinabox** [reply]
https://github.com/dosinabox/g2nr_unofficial_update/blob/master/PrjGOTHIC/AI/Monster/B_Monster/B_MM_AssessOthersDamage.d#L64
